### PR TITLE
sendSMSMessageBird() php implementation

### PR DESCRIPTION
**Pull request by ramdevcm**

### CODE REVIEW

**Comment by ramdevcm** [comment]
@lohanidamodar could you please review this PR

File: php/send-message-bird-sms/readme.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+* Upload your tarfile 
+* Click 'Activate'
+
+## 🎯 Trigger
+
+Output log will be obtained
+
+<div class="center">
+    <img src="OutputLog.png" />
+</div>
```
Comment (Meldiron) [inline php/send-message-bird-sms/readme.md:59]: Let's move screenshot of successful execution out of readme file and lets put it as comment on the PR.

File: php/send-message-bird-sms/readme.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+```
+
+* Navigate to the Overview Tab of your Cloud Function > Deploy Tag
+* Input the command that will run your function (in this case "php index.php") as your entrypoint command
+* Upload your tarfile 
+* Click 'Activate'
+
+## 🎯 Trigger
+
+Output log will be obtained
```
Comment (Meldiron) [inline php/send-message-bird-sms/readme.md:55]: Let's explain how people can trigger the function here - either manually by pressing "execute now" on the function in appwrite console or using `createExecution` in any of our client/server-side SDK

File: php/send-message-bird-sms/readme.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+* **MSG** - Message that is to be sent
+* **APPWRITE_FUNCTION_PROJECT_ID** - Appwrite project ID
+* **RECIPIENT** - Receiver phone number
+* **APPWRITE_API_KEY** - Appwrite API SecretKey
+* **API_END_POINT** - Appwrite EndPoint Eg:- 
+http://localhost/v1
+
+<div class="center">
+    <img src="FunctionVariables.png" />
+</div>
```
Comment (Meldiron) [inline php/send-message-bird-sms/readme.md:18]: We explained it well, let's remove this screenshot.

File: php/send-message-bird-sms/readme.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+A sample php Cloud Function for sending an SMS using message-bird api.
+
+## 📝 Environment Variables
+Go to Settings tab of your Cloud Function. Add the following environment variables.
+
+* **MESSAGE_BIRD_KEY** - Live API Key from Message Bird  
+* **ORIGINATOR** - Sender phone number
+* **MSG** - Message that is to be sent
+* **APPWRITE_FUNCTION_PROJECT_ID** - Appwrite project ID
+* **RECIPIENT** - Receiver phone number
```
Comment (Meldiron) [inline php/send-message-bird-sms/readme.md:11]: Reciever will change in every request. Lets use `APPWRITE_FUNCTION_DATA` for that.

**Review by Meldiron — CHANGES_REQUESTED** [review]
Requested few changes.

**Comment by christyjacob4** [comment]
@ramdevcm  closing this PR for now since there has been no activity in quite a long time. Feel free to reopen it when you're ready to start working on it once again. 🙂

**Pull request closed without merging**